### DISCUSSION TITLE
An Infrared addon board to remote control legacy appliances.

### DISCUSSION THREAD

**Discussion by 256ericpan** [Seeed-Studio/OSHW-XIAO-Series#17] [Open for Vote]
more powerful and encoded 38khz ones, ready to use in the room rather than prototype with https://www.seeedstudio.com/Grove-Infrared-Emitter.html

**Comment by CWGwaltney** [comment]
Yes! I agree... a Grove IR Trancever Module... With Emit and Recieve... and maybe processing power and IIC connection
